package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qrhunt",
-  "version": "1.0.5",
+  "version": "0.0.25",
```

Comment:
Let's leave the version changes to pre-release for testing out of the git commits.